backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_OrderLine.java

```diff
@@ -66,6 +74,13 @@
 	private final OrderGroupCompensationChangesHandler groupChangesHandler;
 
 	public static final String ERR_NEGATIVE_QTY_RESERVED = "MSG_NegativeQtyReserved";
+	private static final String SYS_CONFIG_MAX_HADDEX_AGE_IN_MONTHS = "MAX_HADDEX_AGE_IN_MONTHS";
+	private static final AdMessageKey MSG_HADDEX_CHECK_ERROR = AdMessageKey.of("de.metas.order.model.interceptor.producthadexerror");
```

Comment:
the string from here shall match with the AD_Message.Value.
Else, no message will be found.